packages/devtools_app/lib/src/utils.dart

```diff
@@ -966,7 +966,8 @@ extension LogicalKeySetExtension on LogicalKeySet {
     return sortedKeys.map((key) {
       if (_modifiers.contains(key)) {
         if (isMacOS && key == LogicalKeyboardKey.meta) {
-          return '⌘';
+          // TODO(issue #3352) Switch back to using ⌘ once supported on web.
```

Comment:
See other comments in DevTools for examples of how we link to issues in todos.

Reply:
Done, thanks!